examples/official-storybook/package.json

```diff
@@ -10,7 +10,7 @@
     "graphql": "node ./graphql-server/index.js",
     "image-snapshots": "yarn run build-storybook && yarn run do-image-snapshots",
     "now-build": "node ../../scripts/bootstrap --core && yarn run build-storybook --quiet",
-    "storybook": "cross-env STORYBOOK_DISPLAY_WARNING=true DISPLAY_WARNING=true start-storybook -p 9011 -c ./ -s built-storybooks --no-dll",
```

Comment:
We want `yarn dev` to work with official-storybook so probably leave this one in. The rest can go! 🙏

Reply:
It is so trivial for a PR... but anyway I noticed it :sweat_smile: